cpu/esp32/gen_esp32part.py

```diff
@@ -13,7 +13,7 @@
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
 #
-#     http:#www.apache.org/licenses/LICENSE-2.0
+#     https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Not sure if that poses a problem with both the licence checker script or the license in general, as the license text at https://www.apache.org/licenses/LICENSE-2.0 itself it uses the HTTP version

Reply:
The license checker from Github has no problem with https. Here an example: https://github.com/thotro/arduino-dw1000/blob/master/LICENSE.md

Reply:
I meant our [license checker](https://github.com/RIOT-OS/RIOT/tree/master/dist/tools/licenses) ;-)

Reply:
(it needs to be adapted to pass the CI tests)

Reply:
Thanks for the hint, I will check this :+1: